src/measure_theory/integral/integrable_on.lean

```diff
@@ -246,6 +246,21 @@ begin
   exact ((Lp.mem_ℒp _).restrict s).mem_ℒp_of_exponent_le hp,
 end
 
+lemma lintegral_to_real_le_lintegral_nnnorm (f : α → ℝ) :
+  ∫⁻ x, ennreal.of_real (f x) ∂μ ≤ ∫⁻ x, ∥f x∥₊ ∂μ :=
+begin
+  simp_rw ← of_real_norm_eq_coe_nnnorm,
+  refine lintegral_mono (λ x, ennreal.of_real_le_of_real _),
+  rw real.norm_eq_abs,
+  exact le_abs_self (f x),
+end
+
+lemma integrable_on.lintegral_lt_top_ae' {f : α → ℝ} {s : set α} (hf : integrable_on f s μ) :
+  ∫⁻ x in s, ennreal.of_real (f x) ∂μ < ∞ :=
+calc ∫⁻ x in s, ennreal.of_real (f x) ∂μ
+    ≤ ∫⁻ x in s, ↑∥f x∥₊ ∂μ : lintegral_to_real_le_lintegral_nnnorm f
+... < ∞ : hf.2
```

Comment:
You could have both the version for the full integral and this version for a set integral (and lemmas about set integrals should use `set_lintegral` in the name):
```suggestion
lemma integrable.lintegral_lt_top {f : α → ℝ} (hf : integrable f μ) :
  ∫⁻ x, ennreal.of_real (f x) ∂μ < ∞ :=
calc ∫⁻ x, ennreal.of_real (f x) ∂μ
    ≤ ∫⁻ x, ↑∥f x∥₊ ∂μ : lintegral_to_real_le_lintegral_nnnorm f
... < ∞ : hf.2

lemma integrable_on.set_lintegral_lt_top {f : α → ℝ} {s : set α} (hf : integrable_on f s μ) :
  ∫⁻ x in s, ennreal.of_real (f x) ∂μ < ∞ :=
integrable.lintegral_lt_top hf
```